04-mpu6050/script/mpu6050.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+if [ "$EUID" -ne 0 ]
```

Comment:
Root check is not required because i2c tools can work correctly with users from group i2c.